pkg/controller/fileobserver/observer_default.go

```diff
@@ -0,0 +1,23 @@
+// +build !linux
+
+package fileobserver
+
+import (
+	"github.com/golang/glog"
+)
+
+type defaultObserver struct{}
+
+func init() {
+	NewObserver = func() (Observer, error) {
+		return &defaultObserver{}, nil
+	}
+}
+
+func (*defaultObserver) Run(stopChan <-chan struct{}) {
+	glog.Warning("file observer is not supported on this platform")
```

Comment:
capital case. What about panic?

Reply:
@sttts yeah, I had it there before and I think panic() is the right behavior here... noone should run this outside containers anyway.